tracing-core/src/field.rs

```diff
@@ -202,8 +209,15 @@ pub trait Visit {
 
     /// Records a type implementing `Error`.
     ///
-    /// **Note**: this is only enabled when the Rust standard library is
+    /// <div class="information">
```

Comment:
something fucky is going on here...

![Screenshot from 2020-06-29 12-01-32](https://user-images.githubusercontent.com/1993852/86045343-4e05f100-ba00-11ea-9764-f0ce6030b6ed.png)